offlineimap/folder/Base.py

```diff
@@ -868,6 +871,8 @@ def __syncmessagesto_copy(self, dstfolder, statusfolder):
                 rtime = self.getmessagetime(uid)
                 statusfolder.savemessage(uid, None, flags, rtime)
                 continue
+            elif uid == 0:
+                continue # skip to the next message (ignore) #336
```

Comment:
This would hide the real cause. Let the copy message thread start so it can explode.